src/Components/App/App.js

```diff
@@ -1,9 +1,15 @@
-import logo from './logo.svg';
+
 import './App.css';
+import {Route, Routes} from 'react-router-dom'
 
 function App() {
   return (
     <div className="App">
+      <Routes>
+        <Route path="/" element={<h1>category component</h1>}/>
```

Comment:
These look good for a start!   We can hash out the error handling components when things are a bit more fleshed out.